admin/spec/solidus_admin/main_nav_item_spec.rb

```diff
@@ -3,6 +3,13 @@
 require "spec_helper"
 
 RSpec.describe SolidusAdmin::MainNavItem do
+  def url_helpers(solidus_admin: {}, spree: {})
+    Struct.new(:solidus_admin, :spree).new(
+      solidus_admin.any? ? Struct.new(*solidus_admin.keys).new(*solidus_admin.values) : nil,
+      spree.any? ? Struct.new(*spree.keys).new(*spree.values) : nil
+    )
```

Comment:
nit: you might also use a double here which might be a little more readable:

```rb
double(
  solidus_admin: double(**solidus_admin),
  spree: double(**spree),
)
```

Reply:
I like it. Changed!! Thanks for the suggestion :raised_hands: